js/privatebin.js

```diff
@@ -1765,8 +1766,9 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
             }
 
             // set text
-            Helper.setElementText($plainText, text);
-            Helper.setElementText($prettyPrint, text);
+            var sanitizedText = DOMPurify.sanitize(text, {SAFE_FOR_JQUERY: true})
+            Helper.setElementText($plainText, sanitizedText);
+            Helper.setElementText($prettyPrint, sanitizedText);
```

Comment:
Ah BTW `setElementText` usesu jQueries `text()` method*, which converts all HTML into a text, so we do not even need sanitation here. (it does not really hurt, possibly only affects performance, but is just not needed)

\* and the IE fallback also sanitizes HTML via the `htmlEntities` helper.